src/xdocs/config_javadoc.xml

```diff
@@ -233,6 +233,122 @@ public class TestClass {
       </subsection>
     </section>
 
+    <section name="JavadocBlockTagLocation">
+      <p>Since Checkstyle 8.24</p>
+      <subsection name="Description" id="JavadocBlockTagLocation_Description">
+        <p>
+          Checks that a
+          <a href="https://docs.oracle.com/en/java/javase/11/docs/specs/doc-comment-spec.html#block-tags">
+          javadoc block tag</a> appears only at the beginning of a line, ignoring leading asterisks,
+          white space, and the initial comment delimiter (&#47;**).
```

Comment:
How is this different then leading asterisks?

Reply:
a little. In `/` sign. Removed for clarity.